app/constants/general.py

```diff
@@ -0,0 +1,21 @@
+"""
+General constants used in the calculations
+"""
+
+DAYS_IN_YEAR = 365.25
+
+AVERAGE_HOUSEHOLD_SIZE = 2.69
+
+DEFAULT_SAVINGS_AND_EMISSIONS_RESPONSE = {
+    "average_household_savings": 1000,
+    "average_household_emissions_percentage_reduction": 85,
+}
+
+EMISSIONS_FACTORS = {
+    "electricity_kg_co2e_per_kwh": 0.077,
```

Comment:
For all these constants and assumptions. My main question is how we document where these came from?

Reply:
Agree, the target should be that this is clear to a newcomer to this codebase. In this case these are from an internal EECA dataset, but the ultimate source is a publicly available report from MFE. It's unlikely to be feasible to get this level of documentation in place prior to go-live.

Reply:
@willcattoneeca I think you have already changed the electricity emission factor to 0.1072 in the branch feature-basic-modelling-structure. But, thought highlighting it here

Reply:
Wood emission factor need to be corrected as 0.005 kgCO2e/kWh